### DISCUSSION TITLE
Problem with used space

### DISCUSSION THREAD

**Discussion by alexey-2000** [minio/minio#16831] [Q&A] [answered]
Hi there.

BG: single Minio server with single data disk (ISCSI target to NAS) for storing Veeam's backup.

Questions:
1. Minio console shows that ~10TB used, OS shows - ~25TB used. Why? How to resolve?
2. Several millions of objects. How to reduce them? Are there any recommendations for that (chunk size etc)?

Thanks in advance.
![Screenshot_7](https://user-images.githubusercontent.com/50793387/225624605-5006abb7-80cd-4fa9-8a16-556ee3d07fb2.png)
![Screenshot_8](https://user-images.githubusercontent.com/50793387/225624626-b255941f-9a4b-4939-8690-4f121b43f74d.png)

**Comment by AlexZIX** [comment]
Show the Monitoring - Metrics - Info page.

**Comment by alexey-2000** [comment]
![image](https://user-images.githubusercontent.com/50793387/228469785-d5219dcf-7ada-465c-9c9b-5519e23c15fe.png)
We've cleared some space with Veeam

**Comment by klauspost** [comment, marked as answer]
It it not really clear what you are asking. The dashboard shows total object size. This does not include erasure codes(probably not in your case), metadata and whatever block overhead there is on your file system, nor multiple versions of objects.

Also, you should upgrade your server.
